### PR TITLE
Require "payload" in AET telemetry json schema

### PR DESCRIPTION
**Pull request by jklukas**
This will avoid a pipeline error that @whd reported when sending a message with content `{}`.

Was:
```
Error message from worker: java.lang.ClassCastException: com.fasterxml.jackson.databind.node.MissingNode cannot be cast to com.fasterxml.jackson.databind.node.ObjectNode com.mozilla.telemetry.aet.DecryptAetIdentifiers$Fn.processDesktopTelemetryPayload(DecryptAetIdentifiers.java:241)
```

### CODE REVIEW

File: ingestion-beam/src/test/java/com/mozilla/telemetry/aet/DecryptAetIdentifiersTest.java [hunk 1]
```diff
@@ -221,4 +221,32 @@ public void testErrorOutput() throws Exception {
...
 
+  @Test
+  public void testTelemetryErrorOutput() throws Exception {
+    // minimal test for throughput of a single document
+    ValueProvider<String> metadataLocation = pipeline
+        .newProvider(Resources.getResource("account-ecosystem/metadata-local.json").getPath());
+    ValueProvider<Boolean> kmsEnabled = pipeline.newProvider(false);
+
+    List<String> input = ImmutableList.of("{}", "{\"payload\":{}}",
+        "{\"payload\":{\"ecosystemClientId\":\"foo\",\"ecosystemDeviceId\":\"bar\"}}");
```
Comment (acmiyaguchi) [inline ingestion-beam/src/test/java/com/mozilla/telemetry/aet/DecryptAetIdentifiersTest.java:232]: Another test case might be a non-empty payload with a disallowed field like `{"payload": {"ecosystemAnonId": "foobar"}}` which is sent to the error stream with a redacted value.

Comment (jklukas) [reply ingestion-beam/src/test/java/com/mozilla/telemetry/aet/DecryptAetIdentifiersTest.java]: Good call. Adding this case exposed an error in the schema for telemetry which is now fixed in this PR.
